processor/blockprocessor/block_processor.go

```diff
@@ -20,6 +22,8 @@ import (
 	"github.com/algorand/indexer/util"
 )
 
+const Prefix = "ledger"
```

Comment:
This is failing the build because there's no comment
```suggestion
const prefix = "ledger"
```